src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/stores/ListStore.js

```diff
@@ -526,6 +526,66 @@ export default class ListStore {
         });
     };
 
+    updateSelectionIds(newSelectionIds: Array<string | number>) {
+        const oldSelectionIds = this.selectionIds;
+
+        if (oldSelectionIds === newSelectionIds) {
+            return;
+        }
+
+        const removedValues = oldSelectionIds.filter((x) => !newSelectionIds.includes(x));
+        const addedValues = newSelectionIds.filter((x) => !oldSelectionIds.includes(x));
+        const notLoadedIds = [];
+
+        removedValues.forEach((oldId: string | number) => {
+            this.deselectById(oldId);
+        });
+
+        addedValues.forEach((newId: string | number) => {
+            const row = this.findById(newId);
+
+            if (!row) {
+                notLoadedIds.push(newId);
+
+                return;
+            }
+
+            this.select(row);
+        });
+
+        if (!notLoadedIds.length) {
+            return;
+        }
+
+        const observableOptions = {};
+
+        for (const key in this.observableOptions) {
+            observableOptions[key] = this.observableOptions[key].get();
+        }
+
+        this.setDataLoading(true);
+        const options = {...observableOptions, ...this.options};
+
+        options.selectedIds = newSelectionIds.join(',');
+
+        this.loadingStrategy.load(
```

Comment:
That is also showing the newly selected items in the list immediately, right? Is that the desired behavior in every case? I think that would e.g. also add all selected items in a table view in an unexpected manner. Have you tried that?

Reply:
@danrot thats the case we want, think when the developer doesn't want that he should not reuse the ResourceStore instead clone the ResourceStore or what do you think?

Reply:
We are not even in the `ResourceStore` here, so I guess you wanted to say `ListStore`.

But I think I was mistaken anyway... The `loadingStrategy` really only loads the data, and does nothing else... So that code should be fine :+1: